apps/core/scenes/bookingDetail/menuGroups/Passengers/VisaDetail.js

```diff
@@ -1,67 +1,31 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
-import { StyleSheet, Text, Icon } from '@kiwicom/universal-components';
-import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { VisaDetail } from '@kiwicom/margarita-components';
 
 import type { VisaDetail_data as VisaDetailType } from './__generated__/VisaDetail_data.graphql';
 
 type Props = {|
   +data: ?VisaDetailType,
 |};
 
-type VisaRequiredType = {
+type VisaRequiredType = {|
   +visaRequired: ?boolean,
-};
+|};
 
 const isVisaRequired = (
   passengers: $ReadOnlyArray<?VisaRequiredType>,
 ): boolean => passengers.some(passenger => passenger?.visaRequired);
 
-const VisaDetail = (props: Props) => {
+const Visa = (props: Props) => {
```

Comment:
Also file should be renamed

Reply:
There are also some names like 
```
import { VisaDetail } from '@kiwicom/margarita-components';
import type { VisaDetail_data as VisaDetailType } from './__generated__/VisaDetail_data.graphql';

...
  +data: ?VisaDetailType,

```

With VisaDetail, it should be consistent.

Reply:
Now its `VisaInfoSummary` as its info for all the passengers in the booking, and the component itself called `VisaInfo`